cygnus-ngsi/src/main/java/com/telefonica/iot/cygnus/sinks/NGSIMySQLSink.java

```diff
@@ -384,11 +384,17 @@ public String getValuesForInsert() {
             
                 while (it.hasNext()) {
                     ArrayList<String> values = (ArrayList<String>) aggregation.get((String) it.next());
+                    String value = values.get(i);
+                    if (!value || value.equals("")) {
+                        value = "NULL";
+                    } else {
+                        value = "'" + value + "'";
```

Comment:
what if value is a number?

Reply:
That check was done for Postgresql and Postgis, but not for MySQL:
https://github.com/telefonicaid/fiware-cygnus/pull/1663/files

Reply:
fixed in 938a871a3c56b1353e1b99801fd70aac4c1e630a